test/Atmos/EDMF/helper_funcs/diagnostics_configuration.jl

```diff
@@ -0,0 +1,42 @@
+"""
+    config_diagnostics(driver_config, timeend; interval=nothing)
+
+Returns the state and tendency diagnostic groups
+"""
+
+function config_diagnostics(driver_config, timeend; interval = nothing)
+    FT = eltype(driver_config.grid)
+    info = driver_config.config_info
+    if interval == nothing
+        interval = "$(cld(timeend, 2) + 10)ssecs"
```

Comment:
Does this mean that the default interval is ~half of the simulation? Could we do 1/10th at least?

Reply:
This was @kpamnany's suggestion. It might be nice if we made a struct and internalize this logic (it's just not immediately intuitive). Basically, IIUC, this code sets the interval to output once during, roughly, the middle of the simulation. I believe diagnostics are automatically dumped at the first and last steps, so it should result in ~3 outputs total.